pyaedt/modules/report_templates.py

```diff
@@ -1456,6 +1514,31 @@ def edit_legend(
         ]
         return self._change_property("legend", "legend", props)
 
+    def hide_legend(self, solution_name=True, trace_name=True, variation_key=True, font_height=1):
+        """Hide current Legend
```

Comment:
```suggestion
        """Hide the legend.
```
I couldn't iimagine why "current' would be necessary. If the parameters were defined, perhaps I would see why it might be needed?